CMakeLists.txt

```diff
@@ -2029,6 +2085,52 @@ target_include_directories(cocos2d PUBLIC
     ${CWD}/external/sources/khronos
 )
 
+if(NINTENDO)
+    target_compile_definitions(cocos2d PUBLIC CC_USE_NVN)
+
+    if(${CMAKE_SIZEOF_VOID_P} STREQUAL "4")
+        if(${MSVC_VERSION} LESS 1920)
+            set(NN_SDK_PLATFORM_FOLDER win32-v141)
```

Comment:
What does NN mean?

Reply:
I think it's an internally agreed-upon acronym for `nintendo`. It's been used extensively across the SDK, i.e. libnnSdk.lib